invenio_app_rdm/communities_ui/templates/semantic-ui/invenio_communities/collections/collection.html

```diff
@@ -16,13 +19,42 @@
 
 {%- block page_body %}
 {{ super() }}
-<div class="ui container fluid theme-tertiary page-subheader rel p-10 mr-0-mobile ml-0-mobile">
-  <h1 class="ui container">
-    {{ collection.title }}
-  </h1>
+{% set root_collection=collection[collection.root] %}
+<div class="ui container fluid page-subheader rel p-10 mr-0-mobile ml-0-mobile">
+  <div class="collection-header ui container">
+    <h1 class="ui medium header">
+      {%- if logo -%}
+        <img src="{{ logo }}" width="45" height="45" alt="Collection logo"/>
+      {%- endif -%}
+      <div class="content">{{ root_collection.title }}</div>
+    </h1>
+  </div>
+</div>
+<!-- Breadcrumbs -->
+<div class="ui container mb-0">
+  <div class="ui breadcrumb">
+    <a class="section"
+      href="{{ url_for('invenio_app_rdm_communities.communities_browse', pid_value=community.slug ) }}">
+      {{ _("Browse") }}
+    </a>
+    <i class="angle right icon divider mr-0" aria-hidden="true"></i>
+
+    {% if root_collection.breadcrumbs %}
+    {%- for breadcrumb in root_collection.breadcrumbs -%}
+    <a class="section" href="{{ breadcrumb.link }}">
+      {{ breadcrumb.title }}
+    </a>
+    {% if not loop.last %}
+    <i class="angle right icon divider mr-0" aria-hidden="true"></i>
+    {% endif %}
+    {%- endfor -%}
+    {%- endif -%}
+  </div>
 </div>
+<div class="ui divider mt-0 mb-0" aria-hidden="true"></div>
+<!-- Collection record search -->
 <div id="collection-search"
-  data-invenio-search-config='{{search_app_collection_config(app_id="InvenioAppRDM.CollectionsSearch", endpoint=collection.links.search, hidden_params=[["q", collection.query]]) | tojson }}'
+  data-invenio-search-config='{{search_app_collection_config(app_id="InvenioAppRDM.CollectionsSearch", endpoint=root_collection["links"]["search"], hidden_params=[["q", root_collection["query"]]]) | tojson }}'
```

Comment:
minor: just mentioning here as well we had an IRL discussion that this query can't be in `hidden_params` since we also want to be able to search in a collection using the searchbar in the top.